homework04-semantics/index.html

```diff
@@ -0,0 +1,386 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>Html semantic tags</title>
+   </head>
+   <body>
+      <!-- header -->
+      <header>
+         <h1 style="text-transform: uppercase; padding-left: 40px">
+            HTML semantic tags
+         </h1>
+         <div
+            style="
+               display: flex;
+               justify-content: space-between;
+               align-items: center;
+            "
+         >
+            <nav>
+               <ul
+                  style="
+                     display: flex;
+                     justify-content: space-between;
+                     gap: 30px;
+                     list-style: none;
+                     font-size: 1.5rem;
+                  "
+               >
+                  <li><a href="#header-tag">header</a></li>
+                  <li><a href="#footer-tag">footer</a></li>
+                  <li><a href="#aside-tag">aside</a></li>
+                  <li><a href="#nav-tag">nav</a></li>
+                  <li><a href="#section-tag">section</a></li>
+                  <li><a href="#article-tag">article</a></li>
+                  <li><a href="#figure-tag">figure</a></li>
+               </ul>
+            </nav>
+            <form target="search">
```

Comment:
form not in header 
please check out what header shopuld contain

Reply:
okay, I will check out about header